app/views/user_mailer/welcome_user.html.erb

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <b>Hi <%=@user.name%>,</b>
+    <p>
+      Welcome to openSNP.org – the platform for sharing genotyping raw data.
+      We hope you enjoy using openSNP, can meet some interesting people and have the chance to learn some new things about our genome.
+    </p>
+    <p>
+      To start get started just upload your genotyping raw-data, enter some of your phenotypic variation and start to socialize with other users.</br>
+            <a href="<%=new_user_session_url()%>">To login just visit <%=new_user_session_url()%></a></br>
+            <a href="<%=edit_user_url(@user)%>">To edit your settings visit <%=edit_user_url(@user)%></a>
+    </p>
+    <p>
+      If you need help <a href="<%=faq_url()%>">you can read the FAQ</a> or contact us by replying to this mail.
+    </p>
+    <p>
+      Cheers,
+      your openSNP-team.
+    </p>
+    --<br/>
+    openSNP thanks <a href="https://www.sbgenomics.com/">Seven Bridges</a> –
+    and its biomedical data analysis platform –
+    along with <a href="patreon.com/openSNP">our Patreons</a> for supporting our
+    hosting costs.</br>
+    <a href="https://www.sbgenomics.com">
+      <%= image_tag("http://"+ActionMailer::Base.default_url_options[:host]+"/assets/SBG_logo.png") %>
```

Comment:
line 44 here should take care of it? https://github.com/gedankenstuecke/snpr/blob/master/config/application.rb